test/jdk/jdk/jfr/event/metadata/TestLookForUntestedEvents.java

```diff
@@ -54,7 +54,7 @@ public class TestLookForUntestedEvents {
 
     private static final Set<String> hardToTestEvents = new HashSet<>(
         Arrays.asList(
-            "DataLoss", "IntFlag", "ReservedStackActivation",
+            "DataLoss", "IntFlag", "ReservedStackActivation", "JitRestart",
```

Comment:
Can we create a test? There is an existing test for CodeCacheFull, maybe derive from it to create a test also for the JitRestart event?